src/pre_process/m_patches.fpp

```diff
@@ -57,6 +58,30 @@ module m_patches
     real(kind(0d0)) :: x_centroid, y_centroid, z_centroid
     real(kind(0d0)) :: length_x, length_y, length_z
 
+    enum, bind(c)
+        enumerator :: kLineSegmentPatch = 1
```

Comment:
I'm unsure how this is different or helpful when compared to just declaring these as integer constant parameters in `common/m_constants.fpp`. @henryleberre had something in mind but honestly I'm not familiar with Fortran enumerations.